provision/aws/rds/aurora_common.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+if [ -f ./.env ]; then
+  source ./.env
+fi
+
+export AURORA_CLUSTER=${AURORA_CLUSTER:-"keycloak"}
+export AURORA_INSTANCE=${AURORA_INSTANCE:-"${AURORA_CLUSTER}-instance-1"}
```

Comment:
Looks like there are some constraints for values of `db-instance-identifier`. For example it looks like it is not possible to use `_`. If the default is used here it puts the same constraints on the value of the `AURORA_CLUSTER` variable. Maybe we should put a note in the docs.

Reply:
I've added an explanation about the Aurora cluster naming restrictions. I couldn't find details of restrictions on db-instance naming in [this doc](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html#Aurora.CreateInstance.Settings), the only restriction on underscores I could find was with regards to the DB name.